.github/workflows/auto-comment-pr.yml

```
@@ -23,7 +23,13 @@ jobs:
             // Get PR creator's username
             const prCreator = context.payload.pull_request.user.login;
             
-            const commentBody = `🔊@${prCreator} 🚀Thanks for your contribution 🎉. CodeRabbit(AI) will review your code first 🔥`;
+            const commentBody = `
+            🔊@${prCreator} 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)